nexus-repository-r/src/main/java/org/sonatype/nexus/repository/r/internal/RComponentMaintenance.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Sonatype Nexus (TM) Open Source Version
+ * Copyright (c) 2017-present Sonatype, Inc.
+ * All rights reserved. Includes the third-party code listed at http://links.sonatype.com/products/nexus/oss/attributions.
+ *
+ * This program and the accompanying materials are made available under the terms of the Eclipse Public License Version 1.0,
+ * which accompanies this distribution and is available at http://www.eclipse.org/legal/epl-v10.html.
+ *
+ * Sonatype Nexus (TM) Professional Version is available from Sonatype, Inc. "Sonatype" and "Sonatype Nexus" are trademarks
+ * of Sonatype, Inc. Apache Maven is a trademark of the Apache Software Foundation. M2eclipse is a trademark of the
+ * Eclipse Foundation. All other trademarks are the property of their respective owners.
+ */
+package org.sonatype.nexus.repository.r.internal;
+
+import java.util.Collections;
+import java.util.Set;
+
+import org.sonatype.nexus.common.entity.EntityId;
+import org.sonatype.nexus.repository.storage.Asset;
+import org.sonatype.nexus.repository.storage.Component;
+import org.sonatype.nexus.repository.storage.DefaultComponentMaintenanceImpl;
+import org.sonatype.nexus.repository.storage.StorageTx;
+import org.sonatype.nexus.repository.transaction.TransactionalDeleteBlob;
+import org.sonatype.nexus.transaction.UnitOfWork;
+
+import com.google.common.collect.Iterables;
+
+/**
+ * R Component and asset removing implementation
+ */
```

Comment:
Missed  (at)since 3.next

Reply:
Added since 1.1.0 in b25f344d1f7b239d7ba5a4450970c54137fedd8a